src/main/webapp/i18n/de/course.json

```diff
@@ -87,40 +87,40 @@
             "fileUploadExercises": "Dateiupload-Aufgabe",
             "students": "Studierende",
             "participations": "Teilnahmen",
-            "tutors": "Tutoren",
-            "editors": "Editoren",
-            "instructors": "Dozenten",
-            "student": "Student",
-            "tutor": "Tutor",
-            "editor": "Editor",
-            "instructor": "Dozent",
+            "tutors": "Tutor:innen",
+            "editors": "Editor:innen",
+            "instructors": "Lehrende",
+            "student": "Studierende:r",
+            "tutor": "Tutor:in",
+            "editor": "Editor:in",
+            "instructor": "Lehrperson",
             "form": {
                 "shortname": {
                     "minlength": "Der Kurzname muss mindestens 3 Zeichen lang sein.",
                     "forbidden": "Der Kurzname muss mit einem Buchstaben beginnen und darf keine Sonderzeichen beinhalten."
                 }
             },
-            "courseDashboard": "Kurs Dashboard",
-            "instructorDashboard": "Dozenten-Kursdashboard",
+            "courseDashboard": "Kurs-Dashboard",
+            "instructorDashboard": "Kursdashboard für Lehrende",
             "presentationScoreEnabled": {
                 "title": "Präsentationsbewertung aktiviert",
-                "description": "Aktiviere diese Checkbox und definiere, wie viele Präsentationen ein Student halten muss, um den Notenbonus für die Vorlesung zu erhalten. Nach der Aktivierung kannst du bei der Erstellung jeder Übung definieren, ob dieses Feature dafür aktiviert sein soll."
+                "description": "Aktiviere diese Checkbox und definiere, wie viele Präsentationen Studierende halten muss, um den Notenbonus für die Vorlesung zu erhalten. Nach der Aktivierung kannst du bei der Erstellung jeder Übung definieren, ob dieses Feature dafür aktiviert sein soll."
```

Comment:
```suggestion
                "description": "Aktiviere diese Checkbox und definiere, wie viele Präsentationen Studierende:r halten muss, um den Notenbonus für die Vorlesung zu erhalten. Nach der Aktivierung kannst du bei der Erstellung jeder Übung definieren, ob dieses Feature dafür aktiviert sein soll."
```

Reply:
I wanted to use plural but forgot to adapt "muss" to "müssen". Good catch, I fix it.